drongr/drongr.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Introduction to drongr</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="drongr.css" />
+    <!-- <script src="main.js"></script> -->
+</head>
+<body>
+    <div class="exhibit">
+        <div>
+            <img class="illustrate" src="person_illustration@2x.png" alt="person_illustration"/>
+        </div>
+        <div class="Area photo">
+            <h2 class="title">&nbsp;&nbsp;与大虎共影</h2>
+            <img class="photo1" src="photo_one.jpg" alt="photo1"/>
+            <img class="photo2" src="photo_two.jpg" alt="photo2"/>
+        </div>
+        <div class="Area">
+            <h2 class="title experience">&nbsp;&nbsp;在大虎的经历</h2>
+            <div class="messageItem">
+                <img class="icon" src="icon_start@2x.png" alt="icon1"/>
+                <p class="describe">在2018年4月14日，我成功加入了大虎科技。在此之前，我曾感觉有点落寞，但是我却不曾灰心过。</p>
+            </div>
+            <div class="messageItem">
+                <img class="icon" src="icon_grow_up@2x.png" alt="icon2"/>
+                <p class="describe">在接下来的两个月内，我学到了很多新东西，感受到了自己也是很强的，也终于能hold住了。</p>
+            </div>
+            <div class="messageItem">
+                <img class="icon" src="icon_next@2x.png" alt="icon3"/>
+                <p class="describe">以后的以后，我会按照我的想法继续前行。</p>
+            </div>
+        </div>
+        
+        </div>
+    </div>
+    
+</body>
+</html>
```

Comment:
We have a convention which is every file should end with a new line.

eg.
![image](https://user-images.githubusercontent.com/26423749/41841638-6879cf8c-789b-11e8-980e-670c1387ae43.png)

So, we need a plugin called: [editorconfig](https://editorconfig.org/), and add this plugin in your `VSCode`, write some config files like the following code:

```editorconfig
root = true

[*]
charset = utf-8
end_of_line = lf
indent_style = space
indent_size = 2
insert_final_newline = true
trim_trailing_whitespace = true

[*.md]
trim_trailing_whitespace = false
```

With this, every time after you save your file, it will add a new line to the end of this file automatically.